py/term.cpp

```diff
@@ -117,12 +117,13 @@ Term_mul( PyObject* first, PyObject* second )
 	return BinaryInvoke<BinaryMul, Term>()( first, second );
 }
 
-
+#if PY_MAJOR_VERSION < 3
 static PyObject*
 Term_div( PyObject* first, PyObject* second )
 {
 	return BinaryInvoke<BinaryDiv, Term>()( first, second );
 }
+#endif
```

Comment:
Ditto with `__div__`. We should should be able to divide expressions and terms on Python 3. There should be a different slot to fill on the Py3 type struct.